datafusion/sqllogictest/test_files/parquet.slt

```diff
@@ -169,6 +171,38 @@ SELECT min(date_col) FROM test_table;
 ----
 1970-01-02
 
+# Clean up
+statement ok
+DROP TABLE test_table;
+
+# Do one more test, but order by numeric columns:
```

Comment:
I would like to request moving the tests for this optimization to its own .slt file (e.g. `sorted_parquet.slt` or something). It is important enough and tricky enough I think to warrant specialized tests